src/backend/stream.rs

```diff
@@ -379,14 +379,14 @@ impl<'ctx> PulseStream<'ctx> {
                         s.set_state_callback(check_error, stm.as_mut() as *mut _ as *mut _);
                         s.set_write_callback(write_data, stm.as_mut() as *mut _ as *mut _);
 
-                        let buffer_size_bytes = latency_frames * stm.output_sample_spec.frame_size() as u32;
+                        let buffer_size_bytes = USecExt::to_bytes(latency_frames as u64, &stm.output_sample_spec) as u32;
 
                         let battr = pa_buffer_attr {
                             maxlength: u32::max_value(),
                             prebuf: u32::max_value(),
-                            fragsize: u32::max_value(),
-                            tlength: buffer_size_bytes * 2,
-                            minreq: buffer_size_bytes / 4
+                            fragsize: buffer_size_bytes,
```

Comment:
I don't think this change has any effect. We are configuring playback at this point and `fragsize` is a recording related field.